backend/src/util/data_migrate.py

```diff
@@ -0,0 +1,48 @@
+import json
+import asyncio
+from prisma import Prisma
+
+# Initialize Prisma client
+client = Prisma()
+
+async def main():
+    # Connect to the database
+    await client.connect()
+    
+    # Delete existing records from the Page table
+    await client.page.delete_many()
+
+    # Delete existing records from the Book table
+    await client.book.delete_many()
+
+    # Open and load the JSON file
+    with open('/home/nphu01/CodeKid/codekids/frontend/src/util/books.json', 'r') as file:
```

Comment:
Make this relative so it works for others
```suggestion
    with open('../frontend/src/util/books.json', 'r') as file:
```